spec/models/piece_spec.rb

```diff
@@ -50,12 +50,64 @@
       expect(move.game_id).to eq(@game.id)
     end
 
-    it "should raise an error if the arguments are out of bounds" do
-      piece = @game.pieces.create(
-        row: 0, col: 0, is_captured: false, user: @user
-      )
+    # MeO tests for move_to! - test valid_move? and its called methods
+    from_row = 3
+    from_col = 2
+
+    it "should not move the piece: nil move" do
```

Comment:
Even if move_to! did move to the nil move, this test would still pass, since the nil move and initial position are the same. The test needs to somehow determine that the `else` branch in move_to! was executed for nil moves.